src/components/FeatureViews/Accounts/TabComponents/ParentContact.js

```diff
@@ -5,28 +5,53 @@ import Loading from "components/OmouComponents/Loading";
 import ProfileCard from "../ProfileCard";
 import PropTypes from "prop-types";
 import React from "react";
-import {useSelector} from "react-redux";
+import gql from "graphql-tag";
+import { useQuery } from "@apollo/react-hooks";
 import LoadingError from "./LoadingCourseError";
 
-const ParentContact = ({parent_id}) => {
-	const parentStatus = hooks.useParent(parent_id);
-	const parent = useSelector(({Users}) => Users.ParentList[parent_id]);
-
-	if (!parent) {
-		if (hooks.isLoading(parentStatus)) {
-			return <Loading loadingText="PARENT LOADING" small/>;
+const GET_PARENTS = gql`
+	query getParents($id:ID){
+		parent(userId: $id) {
+		user {
+			email
+			firstName
+			id
+			lastName
+		}
+		accountType
+		phoneNumber
 		}
+	}
+`
 
-		if (hooks.isFail(parentStatus)) {
-			return <LoadingError error="parent"/>;
+
+const ParentContact = ({ parent_id }) => {
+	const { data, loading, error } = useQuery(GET_PARENTS, { "variables": { "id": parent_id } });
+	if (loading) {
+		return <Loading loadingText="PARENT LOADING" small />
+	}
+	if (error) {
+		return <LoadingError error="parent" />;
+	}
+
+	const parent = {
+		"name": `${data.parent.user.firstName} ${data.parent.user.lastName}`,
```

Comment:
use the `fullName()` function from utils.js: `"name": fullName(data.parent.user)`